client/state/ui/layout-focus/reducer.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import includes from 'lodash/includes';
+
+/**
+ * Internal dependencies
+ */
+import {
+	LAYOUT_SET_FOCUS,
+	LAYOUT_NEXT_FOCUS_ACTIVATE,
+	LAYOUT_SET_NEXT_FOCUS,
+	SERIALIZE,
+	DESERIALIZE,
+} from 'state/action-types';
+import layoutFocusModule from 'lib/layout-focus';
+
+const initialState = { current: null, previous: null, next: null };
+
+export default function layoutFocus( state = initialState, action ) {
+	const validAreas = [ 'content', 'sidebar', 'sites', 'preview' ];
+	const isValidArea = area => includes( validAreas, area );
```

Comment:
This helper function could then similarly be moved up. If there's a worry about confusing the function as a reducer, might be appropriate to split off a `utils.js` file. I always feel dirty doing it since it turns into something of a dumping ground, but at least helps alleviate this confusion.